frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/hooks/k8s-watch-types.ts

```diff
@@ -0,0 +1,21 @@
+import { Dispatch } from 'react-redux';
+import { K8sModel } from '../../../api/common-types';
+import { SDKStoreState } from '../../../app/redux-types';
+import { WatchK8sResource } from '../../../extensions/console-types';
+
+export type RootState = any;
```

Comment:
Okay, we need to do something about this unfortunately.

Since in order to fix it, we need that other PR to fix the [impure reducer problem](https://github.com/openshift/console/blob/master/frontend/public/reducers/k8s.ts#L91-L131), we should probably name this `OpenShiftReduxRootState` and slap a `@deprecated` notice on it. This needs to be cleaned up before we can move.

```suggestion
/** @deprecated - this needs to be removed once we internalize all k8s redux state
 * TODO: Delete this and all references in the SDK. Favour SDKStoreState
 */
export type OpenShiftReduxRootState = any;
```

Reply:
Done.